php/example_code/workdocs/CreateFolder.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * This file is licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License. A copy of
+ * the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * For more information about creating a WorkDocs application see the WorkDocs Developer Guide at
+ * https://docs.aws.amazon.com/workdocs/latest/developerguide/wd-auth-user.html
+ *
+ *
+ */
+// snippet-start:[workdocs.php.create_folder.complete]
+// snippet-start:[workdocs.php.create_folder.import]
+
+require 'vendor/autoload.php';
+
+use Aws\Exception\AwsException;
+use Aws\WorkDocs\WorkDocsClient;
+
+// snippet-end:[workdocs.php.create_folder.import]
+
+
+/**
+ * Create a new folder in your Amazon WorkDocs.
+ *
+ * This code expects that you have AWS credentials set up per:
+ * https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/guide_credentials.html
+ */
+
+// Create a workdocs Client 
+// snippet-start:[workdocs.php.create_folder.main]
+$client = new Aws\WorkDocs\WorkDocsClient([
+    'profile' => 'default',
+    'version' => '2016-05-01',
+    'region' => 'us-east-2'
+]);
+
+$auth_token_file_path = 'token.txt';
+$folder_name = 'FolderName'
+$parent_folder = 'parent_folder_id';
+
+try {
+    $file = fopen($auth_token_file, 'r') or die("Unable to open file!");

```

Comment:
* I wouldn't use the `or die` pattern here. If there's an error, it will eventually trigger an exception and be caught in the `catch` block, which is preferable to `die`. You could also check for `$file === false` if you want to make sure to throw an informative exception in that case.
* Also, should this variable be `$auth_token_file_path`, from above?